lightpath/ui/gui.py

```diff
@@ -248,6 +257,7 @@ def change_path_display(self, value=None):
         # Update device type checkboxes
         self.update_device_types()
         self.setWindowTitle(f'Lightpath - {self.selected_beamline()}')
+        self.loading_splash.hide()
```

Comment:
if you select `TXI`, it ends up finding zero devices and the splash loading display gets stuck open forever

Reply:
This is a good catch, we can both handle the opening/closing of the screen better and also not give users the option to open a beamline with no devices/beampath.  Addressed in df288b0